changelog/5227.trivial.rst

```diff
@@ -0,0 +1,2 @@
+Replaced ',' with '&' in fido.search() in following doc files: docs/code_ref/net.rst, docs/guide/acquiring_data/fido.rst, docs/guide/acquiring_data/hek.rst, docs/guide/acquiring_data/jsoc.rst, docs/guide/tour.rst, docs/whatsnew/0.8.rst, docs/whatsnew/0.9.rst, docs/whatsnew/2.0.rst, 
```

Comment:
I don't think there needs to be a list of filenames here, just the first bit of the changelog entry is fine.